packages/taquito-michelson-encoder/src/tokens/ticket-deprecated.ts

```diff
@@ -0,0 +1,106 @@
+import { TicketDeprecatedTokenSchema } from '../schema/types';
+import { IntToken } from './comparable/int';
+import { ContractToken } from './contract';
+import { Token, TokenFactory, Semantic, SemanticEncoding } from './token';
+
+/**
+ *  @category Error
+ *  @description Error that indicates a failure when encoding and sending a ticket to the blockchain
+ */
+export class EncodeTicketError extends Error {
```

Comment:
Did we need a TicketDeprecatedEncodeError instead? In essence, it seems to do the same job as TicketEncodeError just more descriptive to be ticket-deprecated.

Reply:
Yes I think TicketDeprecatedEncodeError would be more descriptive

Reply:
Gotcha, comment addressed in the new commit :)